misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/EvaluationUtils.java

```diff
@@ -36,13 +41,57 @@
  */
 public class EvaluationUtils {
 
-    private static final String JAVA_LANG_CLASS = "java.lang.Class";
+    // Helper classes
+    public static final String B_TYPE_CHECKER_CLASS = "org.ballerinalang.jvm.TypeChecker";
     private static final String B_STRING_UTILS_CLASS = "org.ballerinalang.jvm.api.BStringUtils";
+    public static final String JAVA_BOOLEAN_CLASS = "java.lang.Boolean";
+    public static final String JAVA_LONG_CLASS = "java.lang.Long";
+    public static final String JAVA_DOUBLE_CLASS = "java.lang.Double";
+    public static final String JAVA_LANG_CLASS = "java.lang.Class";
+    public static final String JAVA_OBJECT_CLASS = "java.lang.Object";
+    // Helper methods
+    public static final String GET_TYPEDESC_METHOD = "getTypedesc";
+    public static final String VALUE_OF_METHOD = "valueOf";
+    public static final String REF_EQUAL_METHOD = "isReferenceEqual";
+    public static final String VALUE_EQUAL_METHOD = "isEqual";
```

Comment:
Can these be private? These are used only within the EvaluationUtils class right?

Reply:
@Ibaqu These are being used outside of the class (inside the debugger evaluation engine implementation). `FROM_STRING_METHOD` and `FOR_NAME_METHOD` constants are used only inside the class and hence declared as private.